src/components/modal/ModalDialog.js

```diff
@@ -49,6 +64,24 @@ class ModalDialog extends React.Component<Props> {
 
     modalID: string = uniqueId('modal');
 
+    /**
+     * Renders a button if onBackButtonClick is passed in
+     * @return {ReactElement|null} - Returns the button, or null if the button shouldn't be rendered
+     */
+    renderBackButton() {
+        const { backButtonClick } = this.props;
+        if (!backButtonClick) {
+            return null;
+        }
+
+        return (
+            // eslint-disable-next-line react/button-has-type
+            <button className="modal-back-button" onClick={this.onBackButtonClick}>
```

Comment:
Buttons are `type="submit"` by default, which can cause a page refresh. Can we add `type="button"` to prevent that and also avoid the eslint directive?